flocx_market/objects/offer.py

```diff
@@ -47,3 +47,10 @@ def save(cls, data):
         cls.status = data['status']
         db.offer_update(cls.marketplace_offer_id, data)
         return cls
+
+    @classmethod
```

Comment:
I'd split this into two methods:

* a classmethod that's called `get_all_to_be_expired`
* a method that's called `expire`

Then, update the manager service method in `manager/service.py` to iterate through the first classmethod, and then call the second method on each of its offers. This way, if there's ever, say, a REST API for an owner to expire their offer, then we have the `expire` function ready to go.